op-deployer/pkg/deployer/init.go

```diff
@@ -118,9 +118,9 @@ func Init(cfg InitConfig) error {
 		l2ChainIDBig := l2ChainID.Big()
 		intent.Chains = append(intent.Chains, &state.ChainIntent{
 			ID:                         l2ChainID,
-			BaseFeeVaultRecipient:      common.Address{},
-			L1FeeVaultRecipient:        common.Address{},
-			SequencerFeeVaultRecipient: common.Address{},
+			BaseFeeVaultRecipient:      addrFor(devkeys.BaseFeeVaultRecipientRole.Key(l1ChainIDBig)),
+			L1FeeVaultRecipient:        addrFor(devkeys.L1FeeVaultRecipientRole.Key(l1ChainIDBig)),
+			SequencerFeeVaultRecipient: addrFor(devkeys.SequencerFeeVaultRecipientRole.Key(l1ChainIDBig)),
```

Comment:
Shouldn't these keys be parametrized with the L2 chain ID? Otherwise each L2 chain will have the same recipient addresses on the L1.

Reply:
Make sense, since the withdrawal [network is also "local" by default](https://github.com/ashutoshvarma/optimism/blob/7320df5a31fab29d692f4cd4cba054063ee31612/op-deployer/pkg/deployer/state/deploy_config.go#L40) 

I'll change it